CMakeLists.txt

```diff
@@ -114,3 +115,25 @@ if(FLATBUFFERS_BUILD_TESTS)
        "${CMAKE_CURRENT_BINARY_DIR}")
   add_test(NAME flattests COMMAND flattests)
 endif()
+endif(NOT BIICODE)
+
+IF(BIICODE)
+  INIT_BIICODE_BLOCK()
```

Comment:
I'm not a big fan of distribution specific code in the main cmakelists file.  If this was factored out so that biicode was part of an optionally included file it may not be so bad.  Right now if we start adding rules for each distribution mechanism for the library this could get out of hand pretty quickly.